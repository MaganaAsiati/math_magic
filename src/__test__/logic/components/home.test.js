import React from 'react';
import renderer from 'react-test-renderer';

import Home from '../../../components/home';

it('it should match the snapshot', () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
