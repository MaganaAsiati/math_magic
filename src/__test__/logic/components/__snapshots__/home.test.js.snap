// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should match the snapshot 1`] = `
<div
  className="container-home"
>
  <h1>
    Welcome to our page !
  </h1>
  <p
    className="description-home"
  >
    Mathematical literacy includes learning to read and write different types of mathematical texts as part of purposeful mathematical meaning making. Thus in this article, we describe how learning to read and write mathematical texts (proof text, algorithmic text, algebraic/symbolic text, and visual text) supports the development of students‘ mathematical literacy. Explicit instruction about how to engage with each text type helps to build students‘ awareness of the function of mathematical texts and of how to leverage them to support the doing of mathematics. Teachers and leaders can use this discussion of mathematical text types to organize and conceptualize instruction within a disciplinary literacy orientation.
  </p>
</div>
`;
